webpack.config.ts

```diff
@@ -67,6 +83,12 @@ const base = async (env?: Partial<WebpackEnvParams>): Promise<Configuration> =>
     },
     module: {
       rules: [
+        {
+          test: /\.m?js$/,
+          resolve: {
+            fullySpecified: false
+          }
+        },
```

Comment:
why is it needed? the project does not contain .mjs file and it is not module library either.